actions/terraform/apply/action.yaml

```diff
@@ -53,46 +53,25 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Terraform Install
-      id: install
-      uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd # v3.1.2
+    - name: Terraform Init
+      uses: ./actions/terraform/init
```

Comment:
If this is used from external repositories, will it fail as this is a local reference?

Reply:
Not sure, but good point. I will add the org/repo...